actors/builtin/miner/policy.go

```diff
@@ -81,47 +91,66 @@ var MaxSealDuration = map[abi.RegisteredSealProof]abi.ChainEpoch{
 // This lookback exists so that deadline windows can be non-overlapping (which make the programming simpler)
-// but without making the miner wait for chain stability before being able to start on PoSt computation.
-// The challenge is available this many epochs before the window is actually open to receiving a PoSt.
+// Motivation: A miner can start the WindowPoSt computation without waiting for chain stability. This value cannot be too large since it could compromise the rationality of honest storage (due to WindowPoSt cost assumptions)
 const WPoStChallengeLookback = abi.ChainEpoch(20)
 
-// Minimum period before a deadline's challenge window opens that a fault must be declared for that deadline.
-// This lookback must not be less than WPoStChallengeLookback lest a malicious miner be able to selectively declare
-// faults after learning the challenge value.
+// PARAM_SPEC
+// Minimum period between fault declaration and the next deadline opening.
+// If the number of epochs between fault declaration and deadline's challenge window opening is lower than FaultDeclarationCutoff,
+// the fault declaration is considered invalid for that deadline.
+// Motivation: This guarantees that a miner is not likely to successfully fork the chain and declare a fault after seeing the challenges.
 const FaultDeclarationCutoff = WPoStChallengeLookback + 50
 
+// PARAM_SPEC
 // The maximum age of a fault before the sector is terminated.
-var FaultMaxAge = WPoStProvingPeriod * 14
+// Motivation: This guarantees to clients that a Filecoin miner cannot lose the file for longer than 14 days.
+const FaultMaxAge = WPoStProvingPeriod * 14
 
+// PARAM_SPEC
 // Staging period for a miner worker key change.
-// Finality is a harsh delay for a miner who has lost their worker key, as the miner will miss Window PoSts until
+// Future improvement: Finality is a harsh delay for a miner who has lost their worker key, as the miner will miss Window PoSts until
 // it can be changed. It's the only safe value, though. We may implement a mitigation mechanism such as a second
 // key or allowing the owner account to submit PoSts while a key change is pending.
+// Motivation: This guarantees that a miner cannot choose a more favorable worker key that wins leader elections.
 const WorkerKeyChangeDelay = ChainFinality
 
-// Maximum number of epochs past the current epoch a sector may be set to expire.
-// The actual maximum extension will be the minimum of CurrEpoch + MaximumSectorExpirationExtension
-// and sector.ActivationEpoch+sealProof.SectorMaximumLifetime()
+// PARAM_SPEC
+// A sector extension can be a maximum number of epochs past the current epoch.
+// A sector may be extended multiple times, however, the maximum extension will be the minimum of
+// CurrEpoch + MaximumSectorExpirationExtension and sector.ActivationEpoch+sealProof.SectorMaximumLifetime()
 const MaxSectorExpirationExtension = builtin.EpochsInYear
```

Comment:
**Clarification requested**:
Hey @zixuanzh:
- What is the value that we should choose? 3 months?
- What is motivation for choosing 3 months or 1 year as "maximum" expiration?